library/psa_crypto.c

```diff
@@ -438,7 +441,10 @@ mbedtls_ecp_group_id mbedtls_ecc_group_of_psa( psa_ecc_family_t curve,
             return( MBEDTLS_ECP_DP_NONE );
     }
 }
-#endif /* defined(MBEDTLS_ECP_C) */
+#endif /* defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR) ||
+        * defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_PUBLIC_KEY) ||
+        * defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR) ||
+        * defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_PUBLIC_KEY) */
```

Comment:
Just below, line 456 there is an MBEDTLS_MD_C guard remaining. I checked all the MBEDTLS_xxx_C that this PR aimed to change to MBEDTLS_PSA_BUILTIN_xxx ones, namely MBEDTLS_ECP_C, MBEDTLS_RSA_C, MBEDTLS_MD_C, MBEDTLS_PKCS1_V15,  MBEDTLS_PKCS1_V2, MBEDTLS_ECDH_C and it seems to be the only one remaining.